app/services/foreman/parameters/validator.rb

```diff
@@ -43,6 +45,17 @@ def validate_list
         true
       end
 
+      def validate_hash
+        return true if value.is_a?(Hash)
```

Comment:
Could a YAML or JSON key not have an Array value still?

It's unclear to me if there's any test coverage of this conditional.